cunit/libconfig.testc

```diff
@@ -994,20 +994,6 @@ static void test_deprecated_stringlist(void)
     val = config_getstring(IMAPOPT_TLS_SESSIONS_DB);
     CU_ASSERT_PTR_NOT_NULL(val);
     CU_ASSERT_STRING_EQUAL(val, "sql");
-
-    /* set both names to different values */
-    CU_SYSLOG_MATCH("Option '.*' is deprecated");
-    config_read_string(
-        "configdirectory: "DBDIR"/conf\n"
-        "tlscache_db: sql\n"
-        "tls_sessions_db: zeroskip\n"
-    );
-    CU_ASSERT_SYSLOG(/*all*/0, 1);
-
-    /* should read new value at the new name */
-    val = config_getstring(IMAPOPT_TLS_SESSIONS_DB);
-    CU_ASSERT_PTR_NOT_NULL(val);
-    CU_ASSERT_STRING_EQUAL(val, "zeroskip");
```

Comment:
We need to keep this part of the test.  The point is to make sure the config parser behaves correctly with deprecated a STRINGLIST.  It just so happens that tlscache_db/tls_sessions_db is an example of such a thing, which we're using to exercise the behaviours we're testing.

Instead of removing this chunk, it should be fine to use the value "twoskip" instead of "zeroskip".  "twoskip" is a valid value for these options, and is different from "sql", so the interesting behaviour will still be exercised.